jenkins/spark-premerge-build.sh

```diff
@@ -57,6 +57,10 @@ mvn_verify() {
     $MVN_INSTALL_CMD -DskipTests -Dbuildver=322
     env -u SPARK_HOME $MVN_CMD -U -B $MVN_URM_MIRROR -Dbuildver=330 clean install $MVN_BUILD_ARGS \
       -Dpytest.TEST_TAGS='' -pl '!tools'
+
+    env -u SPARK_HOME $MVN_CMD -U -B $MVN_URM_MIRROR -Dbuildver=330cdh clean install $MVN_BUILD_ARGS \
```

Comment:
can you move this up to by the 321cdh one and just make it:

$MVN_INSTALL_CMD -DskipTests -Dbuildver=330cdh

Reply:
Fixed in the latest commit.